lib/models/project.js

```diff
@@ -311,7 +311,7 @@ class Project {
     if (fs.existsSync(`${targetsPath}.js`)) {
       this._targets = this.require(targetsPath);
     } else {
-      this._targets = require('../../blueprints/app/files/config/targets');
+      this._targets = { browsers: ['last 1 Chrome versions', 'last 1 Firefox versions', 'last 1 Safari versions'] };
```

Comment:
When requiring `../../blueprints/app/files/config/targets`, node would for some reason also try to read from `../../blueprints/app/files/package.json`, which is a *template* file, not a real one. 

Previously this caused no issues, as the only parts with "template tags" (or whatever you call these `<% %>`) was *within* `.name` [here](https://github.com/ember-cli/ember-cli/pull/9972/files#diff-e4c0828d65b51abb30d80befe6fe7870dc36a73dad0083a9dcbb09d48f58e1cfL2). But now we have more conditionals in place, which make this package.json template *invalid* JSON (in the raw format), so node would fail parsing it.

As `targets.js` is a JS file, I also couldn't just simply read it without making node *interpret* it (like with something like `fs.readJSONSync()`), so the only fix I could come up with is to unfortunately just duplicate value here. There is a test covering the output of this, so hopefully the chance of this and the actual targets file diverging is very low! 🤔